crossing-training.lic

```diff
@@ -527,12 +547,22 @@ class CrossingTraining
       data['cast'] = "cast #{moon}"
     end
 
-    release_cyclics if data['cyclic']
+    release_cyclics if data['cyclic']    
 
     return unless prepare?(data['abbrev'], data['mana'], data['symbiosis'])
 
-    find_charge_invoke_stow(@settings.cambrinth, @settings.stored_cambrinth, @settings.cambrinth_cap, @settings.dedicated_camb_use, data['cambrinth'])
-
+    if !@settings.cambrinth_items[0]['name']
+      @settings.cambrinth_items = [{
+        'name' => @settings.cambrinth,
+        'cap' => @settings.cambrinth_cap,
+        'stored' => @settings.stored_cambrinth
+      }]
+    end
+    @settings.cambrinth_items.each_with_index do |item, index|
+      if data['cambrinth'][index][0]
+        find_charge_invoke_stow(item['name'], item['stored'], item['cap'], @settings.dedicated_camb_use, data['cambrinth'][index])
+      end
```

Comment:
I added an `if` here to check whether there's any mana assigned to `data['cambrinth'][index]` before calling `find_charge_invoke_stow` on it.  Without this, I noticed that it was invoking all cambrinth even if there was no mana being used with it (wasn't breaking the scripts, just causing unnecessary roundtime).